static/js/public.js

```diff
@@ -121,3 +121,37 @@ jQuery(document).ready(function ($) {
     emailBody: description + CURRENT_PAGE_URL
   });
 });
+
+/**
+ * Set url hash if hash provided in the url,
+ * or set hash based on the active panel
+ */
+$(function(){
+  $('.mdl-tabs__tab').click(function(){
```

Comment:
Should we call `event.preventDefault()` as part of this function? If not, I would expect that clicking the `<a>` element will cause the page to scroll up to the top. (I haven't tested that yet, though.) Is that the behavior we want?